README.md

```diff
@@ -1,30 +1,21 @@
 # LERF: Language Embedded Radiance Fields
 This is the official implementation for [LERF](https://lerf.io).
 
-**NOTE**: LERF is fully usable, however complete integration with Nerfstudio is not *fully* complete, as it requires a temporary separate branch `lerf-merge` in the Nerfstudio repo. In the coming few weeks this will be removed when code is refactored to more seamlessly support LERF. Remaining TODOs:
-- [ ] Refactor Nerfstudio viewer to allow adding javascript components from within model files (this will remove the need for hard-coding a textbox into the viewer)
-- [ ] Integrate into `ns-render` commands to render videos from the command line with custom prompts
-
 
 <div align='center'>
 <img src="https://www.lerf.io/data/nerf_render.svg" height="230px">
 </div>
 
 # Installation
 LERF follows the integration guidelines described [here](https://docs.nerf.studio/en/latest/developer_guides/new_methods.html) for custom methods within Nerfstudio. 
-
-### 1. Install Nerfstudio From Source
-Follow instructions [at this link](https://docs.nerf.studio/en/latest/quickstart/installation.html) to install Nerfstudio **from source**. Checkout the Nerfstudio branch `lerf-merge`.
-
-### 2. Build the viewer
-**NOTE**: When full integration is complete, this will be unnecessary, however in the meantime we need to build the viewer ourselves to enable text prompts.
-
-Follow the instructions [at this link](https://docs.nerf.studio/en/latest/developer_guides/viewer/viewer_overview.html#installing-and-running-locally) to build the viewer locally. **Make sure you are on the branch `lerf-merge`**, since this has extra code for a textbox in the viewer.
-
-### 3. Install the `lerf` package
-Navigate to this folder and run `python -m pip install -e .` This installs entrypoints for Nerfstudio to use
-
-### 4. Run `ns-install-cli`
+### 0. Install Nerfstudio dependencies
+[Follow these instructions](https://docs.nerf.studio/en/latest/quickstart/installation.html) up to and including "tinycudann" to install dependencies and create an environment
+### 1. Clone this repo
+`git clone https://github.com/kerrj/lerf`
+### 2. Install this repo as a python package
+Navigate to this folder and run `python -m pip install -e .`
+
+### 3. Run `ns-install-cli`
```

Comment:
This isn't true (about `ns-install-cli`); only the LERF package installation (`pip install`) is required for `ns-train` to register the LERF command (per https://docs.nerf.studio/en/latest/developer_guides/new_methods.html) 

Other than that, I agree that running `ns-install-cli` is best practice. 

Also, did you ever figure out why the `pip install git+https://..."` doesn't work? (not that it's critical)